.circleci/config.yml

```diff
@@ -646,6 +646,10 @@ workflows:
           filters:
             branches:
               only:
+                # Only release prototype static assets on the primary
+                # scylla-prototype branch. The branch filtering above on
+                # /scylla-prototype/ covers any branch with
+                # "scylla-prototype" in the name.
                 - scylla-prototype
```

Comment:
Another option is to also build and push (aka release) the prototype static assets on any branch with "scylla-prototype" in the name. I don't have a strong preference either way, so I'm open to whichever works best for folks. My initial thought was that only releasing the prototype static assets on the one "scylla-prototype" branch could avoid a situation where a user is testing the prototype and an engineer's work on a separate "scylla-prototype-foo" branch overwrites the files in the middle of a test. There are other options too, e.g., adding a manual approval before pushing the files to s3.

Reply:
I think deploying only the `scylla-prototype` branch will work. I don't think it's likely we'll have multiple devs actively working on the prototypes, and if so, requiring that all changes get merged back to an integration branch is probably a good thing.

Reply:
Having the flexibility, though, won't hurt anything 😄

Reply:
And just to clarify, these changes will still support the scenario where devs are all working in the main scylla-prototype branch. It adds a little flexibility but doesn't force devs to use multiple scylla-prototype branches.